.secrets.baseline

```diff
@@ -65,7 +65,7 @@
     }
   ],
   "results": {},
-  "version": "0.13.1+ibm.56.dss",
+  "version": "0.13.1+ibm.55.dss",
```

Comment:
Looks like you have an old version of the git submodule locally. This is going backwards. Can you:

1. Ensure that your local primary branch is up to date:
  i. git checkout main
  ii. git pull origin main
  iii. git submodule update --rebase
2. Make sure your dev branch is rebased with remote primary branch:
  i. git checkout fix-s2s
  ii. git pull origin main
3. Run the following command to sync the git submodule with main branch:
  i. git submodule update --rebase

Reply:
PS: The git submodule hook should of caught this (unless your local version pre dates the hook)

Reply:
Yes saw this - usually submodule mix up, should be fixed in later commits